builtin/common/information_formspecs.lua

```diff
@@ -20,6 +20,14 @@ local LIST_FORMSPEC_DESCRIPTION = [[
 		button_exit[5,7;3,1;quit;%s]
 	]]
 
+local LIST_FORMSPEC_MODSLIST = [[
+		size[6,8]
+		label[0,-0.1;%s]
+		tablecolumns[color;tree;text]
+		table[0,0.5;6,7;modslist;%s;0]
```

Comment:
Wouldn't be better to use a `5.8` width? It's more consistent with the other formspecs.

Reply:
Do you mean the width of the formspec window itself or the width of the table?

Reply:
Sorry, meant the table width

Reply:
I am not sure about the consistency with other formspecs, but in any case it looks like consistent with its formspec window and well-centered on that.

Reply:
Yup it is, if you look at [#L9](https://github.com/minetest/minetest/blob/8a50fd2f9e711d06de7f55ba0e9015622535b217/builtin/common/information_formspecs.lua#L9) and [#L17](https://github.com/minetest/minetest/blob/8a50fd2f9e711d06de7f55ba0e9015622535b217/builtin/common/information_formspecs.lua#L17) you'll notice they used the same `.8` width, which for the relevant formspecs sizes gives the same right margin : )